verifier/src/report_body.rs

```diff
@@ -878,4 +889,18 @@ mod test {
             1
         );
     }
+
+    #[test]
+    fn attributes_verifier_error_display() {
+        let attributes = Attributes::from(REPORT_BODY_SRC.attributes);
+        let verifier = AttributesVerifier::new(attributes);
+        let display_string = format!("{}", verifier);
+        let expected = format!(
+            "The {} is {:?}",
+            Attributes::spaced_struct_name(),
+            attributes
+        );
```

Comment:
:wrench: This should probably be using the display version (once the SGX side is available)
```suggestion
        let expected = format!("The {} is {attributes}", Attributes::spaced_struct_name());
```

Reply:
Done.